package.json

```diff
@@ -69,7 +69,7 @@
   "scripts": {
     "knit": "knit",
     "build": "knit build --skip-preflight --verbose",
```

Comment:
Wouldn't we want this for tungsten to ensure we always have the correct schema? Otherwise we run the risk of having changes that are only on collab?